shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -851,10 +860,6 @@ - (FlutterViewController*)viewController {
 }
 
 - (FlutterCompositor*)createFlutterCompositor {
```

Comment:
Is this still the right name for the method? "Create" usually implies allocation of some sort. Here what we're doing is more like initialisation the existing field and returning a pointer to it.

Reply:
I think it's still a reasonable name. For whoever uses `FlutterEngine`, this method creates a  `embedderAPI/FlutterCompositor` object and returns it. How the object is memory-managed is implementation detail.